index.html

```diff
@@ -39,108 +45,156 @@ <h1><span class="first-part">I'm Tonny Tei,</span><br><span class="second_part">
 
     <section id="mid">
         <!-- second page -->
-         <div class="left-block">
-            <div class="primary-image">
-                <img src="images/project1.png" alt="photo_of_my_projects">
-            </div>
-            <div class="title">
-                <h1>Tonic</h1>
+         <div class="left-block" id="second-page">
```

Comment:
- [ ] Kindly ensure that the color, font size, and font weight of all your project description text matches your chosen Figma design 👍🏽 . Kindly follow the design specification on Figma 👍🏽 

| Figma Design  | Your Design |
| ------------- | ------------- |
|![image](https://user-images.githubusercontent.com/63473584/194623752-ffc98468-11f8-47a9-8c74-7914e0f4e8d2.png)  | ![image](https://user-images.githubusercontent.com/63473584/194623921-fe4fc186-8a34-48ed-97a0-a0c4286b0e57.png)  |